woocommerce/woocommerce-android
# Issue/3207 add attribute part2

This PR continues the work towards editing product variation attributes, and focuses on showing the correct data in the attribute options (aka "terms") screen. No actual editing or updating of attributes is available yet. I also added a skeleton view to the "Add Attribute" screen, since that was missing from the previous PR.

To test:

* Go to product detail for a variable product which has variations 
* Tap Variations
* In the variation list, tap the options menu and select "Edit attributes"
* Tap "Add Attribute"
* Tap an existing attribute
* Ensure the top list shows only attribute options that are assigned to the product
* Ensure the bottom list shows only attribute options that are not assigned to the product

![options](https://user-images.githubusercontent.com/3903757/108271201-6fa61b00-713e-11eb-8865-d3eef3e2de78.png)

Update release notes:
